platform/client-admin/src/config/clientConfig.ts

```diff
@@ -74,25 +71,22 @@ const splitLink = ApolloLink.split(
   httpLink,
 );
 
-/**
- * Instantiate cache object
- * and define cache redirect queries
- * 
- */
-const cache = new InMemoryCache({
- });
+const noCache = 'no-cache';
 
-export const clientConfig: ApolloOfflineClientOptions = {
-  link: authLink.concat(splitLink),
-  cache: cache,
-  conflictListener: new ConflictLogger(),
-  networkStatus: new CapacitorNetworkStatus(),
-  inputMapper: {
-    deserialize: (variables: any) => {
-      return (variables && variables.input) ? variables.input : variables;
-    },
-    serialize: (variables: any) => {
-      return { input: variables }
-    }
+const defaultOptions = {
+  watchQuery: {
+    fetchPolicy: noCache,
+  },
+  query: {
+    fetchPolicy: noCache,
+  },
+  mutate: {
+    fetchPolicy: noCache
   }
+}
+
+export const clientConfig = {
+  defaultOptions,
+  cache: new InMemoryCache(),
```

Comment:
Actually if we do not use cache maybe we simply do not pass cache at all.

Reply:
Yes, my initial reaction but then `ts` complained.  Also seems like `apollo-client` will verify that a cache object is defined - worth a PR on their repo? WDYT?

Reply:
Yes. I would do that as issue first and send PR if they like it

https://github.com/apollographql/apollo-client/blob/master/src/ApolloClient.ts#L42

Reply:
What I am thinking is, if `no-cache` can be passed as default option, there is not need to make the https://github.com/apollographql/apollo-client/blob/8ee7648371fb3bac621af5c38fd98afb1f0bd7d7/src/ApolloClient.ts#L42

Reply:
Ah seems like you beat me to the link ha ha

Reply:
Also this condition will need changing I guess https://github.com/apollographql/apollo-client/blob/8ee7648371fb3bac621af5c38fd98afb1f0bd7d7/src/ApolloClient.ts#L140. 

But yes, good thinking. Let's open an issue for that first in their repo.